types/wordpress__blocks/index.d.ts

```diff
@@ -70,6 +76,32 @@ export interface Block<T extends Record<string, any> = {}> {
      * Component to render in the editor.
      */
     readonly edit?: ComponentType<BlockEditProps<T>> | undefined;
+    /**
+     * Block type editor script definition.
+     * It will only be enqueued in the context of the editor.
+     *
+     * @see {@link https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#editor-script}
+     */
+    readonly editorScript?: string;
+    /**
+     * Block type editor style definition.
+     * It will only be enqueued in the context of the editor.
+     *
+     * @see {@link https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#editor-style}
+     */
+    readonly editorStyle?: string;
+    /**
+     * It provides structured example data for the block.
+     *
+     * @see {@link https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#example}
+     */
+    readonly example?: {
+        readonly attributes: {
+            readonly [k in keyof T]: any;
```

Comment:
I have my own questions if the type for `attributes` inside of `Block<T>` is accurate, but here I think we should be able to rely on the `T` more directly. is there a reason we shouldn't use this here instead of `any`?

```ts
readonly [k in keyof T]: T[k];
```

or maybe even a bit clearer,

```ts
readonly attributes: Readonly<T>
```